WalletWasabi.Gui/Controls/WalletExplorer/TransactionBroadcasterView.xaml

```diff
@@ -15,7 +15,7 @@
     <Grid Classes="content">
       <DockPanel LastChildFill="True">
         <StackPanel DockPanel.Dock="Bottom" Orientation="Horizontal" HorizontalAlignment="Right" Spacing="10">
-          <Button Command="{Binding PasteCommand}" Content="Paste From Clipboard" />
+          <Button Command="{Binding PasteCommand}" Content="Paste PSBT or Transaction Hex" />
```

Comment:
I saw this in the other PR but it is not clear to me. "Paste From Clipboard" is absolutely clear but, "Paste PSBT..." is not.

Reply:
but isn't `paste from clipboard` redundant? where else should it be pasted form - it's obviously always the clipboard...

so, the question is not `where from` but `what` is being pasted.

Reply:
People shouldn't need to know about PSBT or Hex so, the best would be to call it `Paste Transaction` (there is the what) but for some reason I think the "From clipboard" help users to understand what the workflow is.

Reply:
> but what is being pasted.

That was my thinking also.
I agree with changing it to `Paste Transaction`.

Reply:
ACK `Paste Transaction`.

Reply:
Done